cmd/kube-apiserver/app/server.go

```diff
@@ -84,49 +84,71 @@ cluster's shared state through which all other components interact.`,
 
 // Run runs the specified APIServer.  This should never exit.
 func Run(s *options.ServerRunOptions) error {
+	config, sharedInformers, err := BuildServer(s)
+	if err != nil {
+		return err
+	}
+
+	return RunServer(config, sharedInformers, wait.NeverStop)
+}
+
+// RunServer uses the provided config and shared informers to run the apiserver.  It does not return.
+func RunServer(config *master.Config, sharedInformers informers.SharedInformerFactory, stopCh <-chan struct{}) error {
```

Comment:
Arg, how much do I hate that Github is folding some files at will. Of course, it is used, but GH doesn't want me to see it.